aspnetcore/getting-started/index.md

```diff
@@ -1,27 +1,46 @@
 ---
 title: Get started with ASP.NET Core
 author: rick-anderson
-description: A quick tutorial that creates and runs a simple Hello World app using ASP.NET Core.
+description: A quick tutorial that creates and runs a simple Hello World app using ASP.NET Core. 
 ms.author: riande
 ms.custom: mvc
+ms.topic: tutorial
```

Comment:
Remove the `ms.topic` metadata from the doc. This is applied globally via *docfx.json*: https://github.com/aspnet/Docs/blob/master/aspnetcore/docfx.json#L83. Currently, `ms.topic: conceptual` is being applied: https://github.com/aspnet/Docs/blob/master/aspnetcore/docfx.json#L83. Feel free to change this to `tutorial`, if that's the desired effect.